docs/autogenerate_OAI_models.md

```diff
@@ -1,22 +1,49 @@
 ## Motivation
-You will only need to use the script to build models when any of the GA4GH API's openapi specs are updated. The models are being used to return the result of any API call in the correct format, in the correct structs
+You will only need to use the script to build models when any of the GA4GH API's OpenAPI specs are updated. The models are being used to return the result of any API call in the correct format, in the correct structs.
 
+## Prerequisites and Dependencies:
+1. The `script.sh` should have permission to run, which can be done by:
+    ```sh
+    sudo chmod +x ./utils/script.sh
+    ```
+2. OpenAPI Generator CLI: Make sure the directory `~/bin/openapitools` is empty, as OpenAPI is being installed in this script, which might conflict with your existing directory.
```

Comment:
And this is (one reason) why it's better not to install it in the script.

Btw, there is a Docker container available for OpenAPI Generator CLI: https://hub.docker.com/r/openapitools/openapi-generator-cli

You could also refer to that one - this avoids the need for installing anything manually or via script.

Reply:
What do you think about keeping the current as it is (since it isn't anyways going to be used for some time), and making an issue for changing it to this?

Reply:
issue added https://github.com/elixir-cloud-aai/ga4gh-sdk/issues/43